app/api.py

```diff
@@ -297,6 +301,37 @@ def get(self):
             self, ApiActionLog.READ, len(records), len(notes))
 
 
+class PhotoUpload(utils.BaseHandler):
+    https_required = True
+
+    def post(self):
+        if not (self.auth):
+            self.info(
+                403,
+                message='Missing or invalid authorization key',
+                style='plain')
+            return
+        photo = self.request.body_file
+        if photo is not None:
+            photo = photo.read()
+            photo = images.Image(photo)
+            photo, photo_url = create_photo(photo, self)
```

Comment:
This function can throw SizeTooLargeError and PhotoError. Can you catch the exceptions and output appropriate error message?